common/zoom/index.ts

```diff
@@ -0,0 +1,36 @@
+import jwt from 'jsonwebtoken';
+
+export enum MeetingRole {
+    PARTICIPANT = 0,
+    HOST = 1,
+}
+
+type JWTPayload = {
+    appKey: string;
+    sdkKey: string;
+    mn: number;
+    role: MeetingRole;
+    iat: number;
+    exp: number;
+    tokenExp: number;
+};
+
+const generateMeetingSDKJWT = async (meetingNumber: number, role: MeetingRole) => {
+    const iat = Math.round(new Date().getTime() / 1000) - 30;
```

Comment:
why -30? To make sure issued at is always in the past even if server time is off?

Reply:
@timidak can you answer this? Apparently you built this?

Reply:
Correct!